resources/views/notifications/markdown/asset-acceptance.blade.php

```diff
@@ -13,6 +13,9 @@
 @if (isset($declined_date))
 | **{{ ucfirst(trans('general.declined')) }}** | {{ $declined_date }} |
 @endif
+@if (isset($note))
+| **{{ trans('general.declined_note') }}** | {{ $note }} |
```

Comment:
When this is being stored and sent this should just be `Note` I think instead of `Declined Notes`